src/components/deck-gl-wrapper/deck-gl-wrapper.tsx

```diff
@@ -268,8 +268,12 @@ export const DeckGlWrapper = ({
   });
   const [terrainTiles, setTerrainTiles] = useState({});
   const [uvDebugTexture, setUvDebugTexture] = useState(null);
+  const uvDebugTextureRef = useRef(null);
+  uvDebugTextureRef.current = uvDebugTexture;
```

Comment:
`showDebugTextureRef` doesn't work same way?
```
const showDebugTextureRef = useRef<boolean>(showDebugTexture);
showDebugTextureRef.current = showDebugTexture;
```

Reply:
Yes, they re identical.
Do you mean moving showDebugTexture reference assignment from useEffect to line 276?
I agree, it would be better.

Reply:
Yes